components/accordion/index.css

```diff
@@ -120,21 +236,81 @@ governing permissions and limitations under the License.
 
       position: absolute;
       inset-inline-start: 0;
-      inset-block-start: calc(-1 * var(--spectrum-accordion-item-border-size));
+      /*inset-block-start: calc(-1 * var(--spectrum-accordion-item-border-size));
       inset-block-end: calc(-1 * var(--spectrum-accordion-item-border-size));
 
-      inline-size: var(--spectrum-accordion-item-border-left-size);
+      inline-size: var(--spectrum-accordion-item-border-left-size);*/
     }
   }
 }
 
 .spectrum-Accordion-itemContent {
-  padding-block: 0 var(--spectrum-accordion-item-content-padding);
+  /*padding-block: 0 var(--spectrum-accordion-item-content-padding);
   padding-inline: var(--spectrum-accordion-item-content-padding)
-    var(--spectrum-accordion-item-content-padding);
+    var(--spectrum-accordion-item-content-padding);*/
   display: none;
 }
 
+.spectrum-Accordion-item {
+  /*border-color: var(--spectrum-accordion-border-color);*/
+}
+
+.spectrum-Accordion-itemIndicator {
+  /*color: var(--spectrum-accordion-icon-color);*/
+}
+
+.spectrum-Accordion-itemHeader {
+  color: var(--mod-accordion-text-label-color-default, var(--spectrum-accordion-text-label-color-default));
+
+  &:hover {
+    color: var(--mod-accordion-text-label-color-hover, var(--spectrum-accordion-text-label-color-hover));
+    background-color: var(--mod-accordion-background-color-hover, var(--spectrum-accordion-background-color-hover));
+
+    + .spectrum-Accordion-itemIndicator {
+      color: var(--mod-accordion-text-label-color-hover, var(--spectrum-accordion-text-label-color-hover));
+    }
+  }
+
+  &:focus-ring {
+    &:after {
+      background-color: var(--mod-accordion-text-label-color-key-focus, var(--spectrum-accordion-text-label-color-key-focus));
+    }
+  }
+}
+
+.spectrum-Accordion-item {
+  &.is-open {
+    .spectrum-Accordion-itemHeader {
+      &:hover {
+        background-color: var(--mod-accordion-background-color-hover, var(--spectrum-accordion-background-color-hover));
+      }
+    }
+  }
+}
+
+.spectrum-Accordion-item.is-disabled {
+  .spectrum-Accordion-itemHeader {
+    &,
+    &:hover,
+    &:focus-ring {
+      color: var(--mod-accordion-text-label-disabled-color, var(--spectrum-accordion-text-label-disabled-color));
+      background-color: transparent;
+    }
+
+    + .spectrum-Accordion-itemIndicator {
+      color: var(--mod-accordion-text-label-disabled-color, var(--spectrum-accordion-text-label-disabled-color));
+    }
+  }
+}
+
+@media (forced-colors: active) {
+  .spectrum-Accordion-itemHeader {
+    &:focus-ring {
+      outline: 3px solid CanvasText;
+    }
+  }
+}
```

Comment:
I realize that this is probably brought over from the old `skin.css` file, but I wanted to mention that my general approach for working on these migrations is to do all of the other styling first, and then work on the `forced-colors: active` (high-contrast mode) styles _last_. I find it to be easier to approach, and a little easier to think about that way.